modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -744,7 +744,7 @@ public class ApiClient {
      */
     public boolean isJsonMime(String mime) {
       String jsonMime = "(?i)^(application/json|[^;/ \t]+/[^;/ \t]+[+]json)[ \t]*(;.*)?$";
-      return mime != null && mime.matches(jsonMime);
+      return mime != null && (mime.matches(jsonMime) || mime.equals("application/json-patch+json"));
```

Comment:
@brendandburns shall we use a case insensitive comparison with "application/json-patch+json"?

Ref: https://stackoverflow.com/a/12869287/677735